### ISSUE TITLE
responseTimeout claims to be ms, but is probably seconds

### ISSUE THREAD

**Issue by tsteffek** [pytorch/serve#1984]
### 📚 The doc issue

https://pytorch.org/serve/configuration.html tells us: 

```
responseTimeout: the timeout in msec of a specific model’s response. This setting takes priority over default_response_timeout which is a default timeout over all models
```

(notice the milliseconds)
while for `default_response_timeout` it says:

```
default_response_timeout: Timeout, in seconds, used for all models backend workers before they are deemed unresponsive and rebooted. Default: 120 seconds.
```

In the corresponding example, `responseTimeout` is set to 120. We actually have that 120 "ms" limit set in our experimental setup right now, which runs on such a horrible CPU that I'm certain it wouldn't be able to finish in 120 ms.

So while I didn't verify the code (because anyone else will probably find the relevant position way faster) I believe the documentation is wrong here and `responseTimeout` is actually using seconds as well. Even if I'm wrong: I still believe that you probably shouldn't use different scales for two settings with a "default-specific" relationship and I'm sure you can see why.

### Suggest a potential alternative/fix

(After verifying whether my assumption is right)

Change the documentation to reflect that `responseTimeout` is using seconds instead of msec. Maybe also bringing the descriptions in line (as long as they really behave as similarly as I think they do):
```
responseTimeout: Timeout, in seconds, used for this model's backend workers before they are deemed unresponsive and rebooted. This setting takes priority over default_response_timeout which is a default timeout over all models.
```

**Comment by msaroufim** [comment]
You are correct https://github.com/pytorch/serve/blob/master/frontend/server/src/main/java/org/pytorch/serve/wlm/WorkerThread.java#L189

Would you like to make the PR to go ahead and fix this?
